pytorch3d/csrc/nearest_neighbor_points/nearest_neighbor_points.cu

```diff
@@ -219,7 +219,7 @@ at::Tensor nn_points_idx_cuda(at::Tensor p1, at::Tensor p2) {
   const auto D = p1.size(2);
 
   AT_ASSERTM(p2.size(2) == D, "Point sets must have same last dimension.");
-  auto idx = at::empty({N, P1}, p1.options().dtype(at::kLong));
+  auto idx = at::empty({N, P1}, p1.options().dtype(at::kint64_t));
```

Comment:
The Circle CI build is failing as `kint64_t`  is not a valid torch variable. Was `at::kLong` causing an issue?

see: https://pytorch.org/cppdocs/api/variable_namespacetorch_1aca5be74e0c90ee0b02abcdf224231c30.html?highlight=klong

Reply:
Ah. Sorry this should be kept as it is.